File: src/Workflow.Domain/Forms/IT/Item.cs
/**
*@author : Phanny
*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Workflow.Domain.Entities.IT
{
    public class Item
    {
        public int Id { get; set; }
        public int SessionId { get; set; }
        public virtual Session Session { get; set; }
        public string ItemName { get; set; }
        public string Description { get; set; }
        public bool HodRequired { get; set; }
    }
}

File: src/Workflow.Web.Service/Workflow/app/model/itRequestForm/Item.js
Ext.define('Workflow.model.itRequestForm.Item', {
    extend: 'Workflow.model.Base',
	
    fields: [
        {name: 'id',     		type: 'int'		},
        {name: 'sessionId',		type: 'int'		},
        {name: 'itemName',     	type: 'string'	},
		{name: 'description',	type: 'string'		},
		{name: 'hodRequired'	}
    ]
});
